WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController.m

```diff
@@ -1337,8 +1360,15 @@ - (void)configureCell:(UITableViewCell *)cell atIndexPath:(NSIndexPath *)indexPa
     if ([cell isKindOfClass:[QuickStartListTitleCell class]]) {
         ((QuickStartListTitleCell *) cell).state = row.quickStartTitleState;
     }
+
+    if ([cell isKindOfClass:[BloggingRemindersCell class]]) {
```

Comment:
Rather than setting the values at the top of the method and then clearing them here, could we move this condition to the top of the method and just `return` if it's true? (to avoid setting them in the first place)